analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -625,7 +625,7 @@ def signal_handler(*arg, **kwarg):
                            callback=lambda results: worker_result_handler(
                                results, metadata, output_path,
                                context.analyzer_binaries)
-                           ).get(float('inf'))
```

Comment:
Python 3 will throw an exception for inf, but not providing a timeout will result in no timeout exception.

Reply:
Was this just a "temporary" solution because of some problem in the map_async in python2? If it works without is we can remove it.

Reply:
I do not recall any reason why was this there. The tests pass in Python 2, so I would vote for removing this argument.

Reply:
Did you try to kill the pool during the analysis? Every process did stop? I would keep until we fully switch to a python3 interpreter see the [stackoverflow discussion]( https://stackoverflow.com/questions/11312525/catch-ctrlc-sigint-and-exit-multiprocesses-gracefully-in-python) and the [python bug](https://bugs.python.org/issue8296)

Reply:
I see. In this case we could either store the resulting object and pass an argument conditionally or we could always pass a very big timeout like one year which will not throw an exception in python 3 (the exception says inf is too big to be converted to a timestamp).

Reply:
Setting a large timeout should be fine, we can remove it later when only py3 is supported.